### PR TITLE
Fix #5230: VectorDrawableCompat Lint error

### LINKED ISSUE #5230: VectorDrawableCompat
Fix lint warnings related to VectorDrawableCompat

### PR DESCRIPTION
**Pull request by deonwaju**
## Explanation
Fix #5230: VectorDrawableCompat Lint error

Warning: VectorDrawableCompat
Fix: opened the app's build.gradle file.
Inside the android block, added the vectorDrawables section under defaultConfig with the code below:
android { ... defaultConfig { ... vectorDrawables { useSupportLibrary true } } }

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

### CODE REVIEW

**Comment by deonwaju** [comment]
@adhiamboperes PTAL

**Comment by deonwaju** [comment]
This test is the reason for the fail "testStartOverExploration_logsStartCardAndStartOverEvents()" in the "ExplorationProgressControllerTest.kt" file.

**Comment by deonwaju** [comment]
Good day @adhiamboperes ,  
Please, I need help here. So I can move on to the next tasks.

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,5 @@ bazel-*
 .bazelproject
 .aswb
 *.pb
+gradle.properties
+gradle-wrapper.properties
```
Comment (adhiamboperes) [inline .gitignore]: Please revert this change

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @DeonWaju! 

I re-ran the CI, it seems like a flaky test and has now passed.

File: .gitignore (hunk 1 above)
Comment (deonwaju) [reply .gitignore]: Oh, Thank you very much. Ok, I will revert it immediately

Comment (deonwaju) [reply .gitignore]: I have reverted it @adhiamboperes

**Comment by deonwaju** [comment]
@adhiamboperes PTAL

File: .gitignore (hunk 1 above)
Comment (adhiamboperes) [reply .gitignore]: It still shows 2 files changed. I'm only expecting to see app's build.gradle modified.

Comment (deonwaju) [reply .gitignore]: ok, checking

Comment (deonwaju) [reply .gitignore]: Hi @adhiamboperes, I have tried to revert the change, but i am still having two file changes as you said. Even used rebase. I might have to advise to use a new PR branch, if you agree.

**Comment by adhiamboperes** [comment]
> @adhiamboperes PTAL


@DeonWaju, it may be best to open a replacement PR. When creating a commit, only git add the file that has the desired changes, instead of `git add .`. This should be simple to do from within the android studio commit toolbar/vcs.

File: .gitignore (hunk 1 above)
Comment (adhiamboperes) [reply .gitignore]: @DeonWaju, it may be best to open a replacement PR. When creating a commit, only git add the file that has the desired changes, instead of git add .. This should be simple to do from within the android studio commit toolbar/vcs.

Comment (deonwaju) [reply .gitignore]: Ok, I understand, thanks very much
